src/aliceVision/mvsData/imageIO.hpp

```diff
@@ -31,6 +42,30 @@ enum class EImageColorSpace
   NO_CONVERSION
 };
 
+
+struct OutputFileColorSpace
+{
+    EImageColorSpace from = EImageColorSpace::LINEAR;
```

Comment:
EImageColorSpace from{EImageColorSpace::LINEAR};
EImageColorSpace to{EImageColorSpace::AUTO};